upup/pkg/fi/cloudup/spotinsttasks/launch_spec.go

```diff
@@ -157,8 +158,12 @@ func (o *LaunchSpec) Find(c *fi.CloudupContext) (*LaunchSpec, error) {
 
 	// Image.
 	{
+		//		convert spec from api that reply for multi arch data only in spec.images
+		if spec.Images != nil && len(spec.Images) > 1 {
+			spec.SetImageId(fi.PtrTo(fi.ValueOf(spec.Images[0].ImageId)))
+			actual.OtherArchitectureImages = append(actual.OtherArchitectureImages, fi.ValueOf(spec.Images[1].ImageId))
```

Comment:
Nit: Do you want to loop over these other images, so we support more than just 2 in future?